Source/WebCore/PAL/pal/text/EncodingTables.cpp

```diff
@@ -8626,7 +8626,7 @@ const std::array<UChar, 23940>& gb18030()
             (*array)[pointer] = icuOutput;
         }
         
-#if U_ICU_VERSION_MAJOR_NUM < 74
+#if U_ICU_VERSION_MAJOR_NUM < 74 && !HAVE(SYSTEM_ICU_74)
```

Comment:
How about using `ICU::majorVersion()` runtime check? Instead of header's version, it can return actual library's version.